web-marketplace/src/components/organisms/UserAccountSettings/UserAccountSettings.tsx

```diff
@@ -0,0 +1,147 @@
+import { Subtitle, Body } from 'web-components/lib/components/typography';
```

Comment:
could you enable your IDE linter, it looks like there are some linting issues in this file